server/app.js

```diff
@@ -56,6 +60,23 @@ if (process.env.NODE_ENV === "production") {
   });
 }
 
+// GET request from client > server > S3 bucket
+app.get('/profilePhotos/:key', (req, res) => {
```

Comment:
GET requests don't have a request body, you can send the key in the request body of a POST